rainbows.rb

```diff
@@ -27,3 +37,8 @@ def mix_colors(color1,color2)
 
 new_color = mix_colors(red,green)
 puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+rainbow.length.times do |element|
+   color_choice = rainbow[element].keys.join
+   puts Rainbow("#{color_choice}").color(color_choice.to_sym)
```

Comment:
nice. I don't know what join does and I'm too lazy to look it up at this second, but it if works, this looks good!